samples/snippets/csharp/VS_Snippets_Remoting/NclSslClientSync/CS/clientsync.cs

```diff
@@ -40,7 +40,7 @@ public static void RunClient(string machineName, string serverName)
             //<snippet4>
             // Create a TCP/IP client socket.
             // machineName is the host running the server application.
-            TcpClient client = new TcpClient(machineName,443);
+            TcpClient client = new TcpClient(machineName,8080);

```

Comment:
Why do we change the port? 443 is default for ssl/tls while 80 and 8080 is typically used plain HTTP. 
If this is a problem, perhaps the example should take the port as parameter.

Reply:
Sorry for the late back. If you look at the server-side will see there is a port over there also. The point is these two ports must be the same. I didn't choose the 443 to avoid the misunderstanding of using the standard https port number.

Reply:
> Why do we change the port? 443 is default for ssl/tls while 80 and 8080 is typically used plain HTTP. 
> 
> If this is a problem, perhaps the example should take the port as parameter. 

I do agree with the possible confusion using 80 plain HTTP related port.

As mentioned by @DavidEshtehari the server example is using 8080. Should we change both of them?

Using default port can also be good from my point of view.

We should only take care of that, perhaps those code snippets are used more than once and that kind of mismatch could be perhaps found somewhere else.

Reply:
I like the idea of updating the server. Since the client uses the default port, it would just work and be representative of users actually need to do in most cases. And yes, it make sense that the examples work together e.g. use same port.